Automattic/jetpack
# Components: Add `useBreakpointMatch` hook

Insert a new hook, that will help us to identify greater, less, or current breakpoint.

This is useful for cases that we want to hide or do specifics UI changes based on breakpoint.

#### Changes proposed in this Pull Request:

* Add `useBreakpointMatch` hook.
* Add `breakpoints.module.scss` which will be our main source for breakpoints values.

#### Jetpack product discussion


1200295095943283-as-1202214413756002/f

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Storybook`
* Navigate to `JS Packages/Components/Layout/Breakpoint Match`
* Start to play with window size and check if the infos are matching.

#### Demo

https://user-images.githubusercontent.com/1663717/167041217-6fbecc8f-a7e0-417c-84a7-7e58944fb8a9.mp4
